tutorials/5 All QRenderers/5.1 Ansys/Ansys QRenderer.ipynb

```diff
@@ -290,12 +309,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "To **select** an existing design, means to activate an Ansys design and connect to it. You can follow one of three ways:\n",
-    "* re-running the `fourq_hfss.connect_ansys(*with parameters*)`, this time specifying which design to connect to (see section 2.3)\n",
+    "To **select** an existing design, you can follow one of three ways:\n",
```

Comment:
The title is not completely accurate since the activate commands will add a design if it is not there.  

Maybe, better to add text to the section what adds a design.

Reply:
That would be confusing.
The activate methods are to select a design. The creation is just the result of how we decide to handle an exception in the method. We need to keep the 'main scope' of the methods in light, not the exception handling (they can read that from the docs)